google-cloud-spanner/lib/google/cloud/spanner/v1/spanner_client.rb

```diff
@@ -430,18 +430,21 @@ def create_session \
           #
           #   spanner_client = Google::Cloud::Spanner::V1::SpannerClient.new
           #   formatted_database = Google::Cloud::Spanner::V1::SpannerClient.database_path("[PROJECT]", "[INSTANCE]", "[DATABASE]")
-          #   response = spanner_client.batch_create_sessions(formatted_database)
+          #
+          #   # TODO: Initialize `session_count`:
+          #   session_count = 0
+          #   response = spanner_client.batch_create_sessions(formatted_database, session_count)
 
           def batch_create_sessions \
               database,
+              session_count,
```

Comment:
@quartzmo This is a breaking change, yes?

Reply:
dagnabbit wtf

Reply:
It was decided to let this through in the low-level API.

Reply:
SGTM, I'll start now on a PR to update the manual layer for this change.